app/views/categories/index.html.erb

```diff
@@ -0,0 +1,36 @@
+<% content_for :title, "Categories" %>
+<section class="d-flex justify-content-center align-items-center">
+  <div id="categories" class="container text-center">
+    <div class="row">
+      <% if @categories.any? %>
+        <% @categories.each do |category| %>
+          <div class="col-md-6 col-lg-4 mb-4">
+            <%= link_to category, class: 'text-decoration-none' do %>
+              <div class="card bg-light h-100">
+                <div class="card-body">
+                  <div class="d-flex align-items-center">
+                    <img src="<%= asset_path(category.icon) %>" class="card-img-icon me-3">
+                    <div>
+                      <h5 class="card-title mb-0"><%= category.name %></h5>
+                      <small class="text-muted"><%= category.created_at.strftime("%b %d, %Y") %></small>
+                    </div>
+                    <div class="ms-auto">
+                      <span class="text-dark"><%= number_to_currency(category.expenses.sum(:amount)) %></span>
+                    </div>
+                  </div>
+                </div>
+              </div>
+            <% end %>
+          </div>
+        <% end %>
+      <% else %>
+        <div class="col-12 text-white">
+          <p>No categories yet</p>
+        </div>
+      <% end %>
+      <div class="mb-4">
+        <%= link_to "New Category", new_category_path, class: 'btn btn-outline-dark btn-lg' %>
+      </div>
+    </div>
+  </div>
```

Comment:
- [ ] [REQUIRED] You have done a great job here so far 🥇 , however, we need to stick to the design template. Take a  look at the difference between the UI designs below. Kindly fix the "New Category" Button to the bottom of the page and implement the button color as provided in the template. Let's do the same for the Navigation bar on the top of the screen as well. Please, let's implement the same with the transaction page.

Your Implementation   |  Figma Design
:-------------------------:|:-------------------------:
<img width="298" alt="Screenshot 2023-08-18 at 22 22 34" src="https://github.com/naanmohammed/Budget-App/assets/58692590/19b2caed-a414-47bd-b3b8-b2905d540a56"> | <img width="289" alt="Screenshot 2023-08-18 at 22 25 44" src="https://github.com/naanmohammed/Budget-App/assets/58692590/16ca9728-9b7b-499c-a504-5ff274d15b03">